db/migrate/20231123111357_create_custom_field_sections.rb

```diff
@@ -28,9 +28,19 @@ def create_and_assign_default_section
       name: "Project attributes"
```

Comment:
I think it would be a good idea to get rid off the last piece of model code in this migration too and create the section via plain SQL too. This is probably a super simple query and as opposed to this model code it will keep working even if the model should disappear one day.

I know this is outside of the actual scope of the PR but we might as well do it while we're at it.